docs/source/howto/installation_guide.rst

```diff
@@ -51,7 +51,12 @@ which will install pyAFQ locally in editible mode and include extensions.
 .. note::
 
   It is also recommended that you utilize python virtual environment and
-  package mangagement tools and begin with a clean environment.
+  package mangagement tools (e.g. conda) and begin with a clean environment.
+
+.. note::
+
+  You should also install the following optional packages before working through
+  the examples: jupyter, fury, plot, AFQ-browser, matplotlib, pingouin, cloudknot, docker
```

Comment:
Should this be the following?
```suggestion
  the examples: jupyter, fury, plotly, AFQ-browser, matplotlib, pingouin, cloudknot, docker
```

I love pingouin! But are we actually using it in any of the examples?

Reply:
Sorry, one more comment: docker is not a dependency of pyAFQ (although you can run it with docker)

Reply:
@arokem  pyafq[plot] installs pingouin, seaborn, and ipython, which is used in some of the examples. plotly is already installed by default!